### DISCUSSION TITLE
Adding another local storage?

### DISCUSSION THREAD

**Discussion by gluckgluckwasserbauch** [nextcloud/all-in-one#2280] [Questions] [answered]
Hey!

Is it possible to add another external drive?
In my initial install, I forgot to add my 1 TB hard drive. Therefore, all data went to my smaller OS drive.
I had to reinstall my NC setup to make the bigger drive accessible.

Is there a more sustainable solution to add more drives in the future? Maybe by adding or editing some script which includes "mount drive xy"?
I plan to add another 1 TB in a few months.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-allow-the-nextcloud-container-to-access-directories-on-the-host

Apart from this you could look into solutions like mergerFS and put the datadir onto such a mount.
